CHANGELOG.md

```diff
@@ -1,3 +1,36 @@
+## v1.7.0 (02/11/2020)
+
+**olp-sdk-core**
+
+- Added the `sentWith` query parameter to each request.
+- Updated dependencies.
+
+**olp-sdk-authentication**
+
+- Added the `sentWith` query parameter to each request.
+- Updated dependencies.
+
+**olp-sdk-fetch**
+
+- Updated dependencies.
+
+**olp-sdk-dataservice-api**
+
+- Added generated functions and interfaces that support Authorization APIs. You can find them at `@here/olp-sdk-dataservice-api/lib/authorization-api-v1.1.ts`.
+- Added the `Content-Encoding` header to the PUT blob request. Without this, it was not possible to upload gzipped blobs.
+- **Breaking change** Removed deprecated items that have exceeded the six-month deprecation period. @see #385.
```

Comment:
```suggestion
- **Breaking change** Removed deprecated items that have exceeded the six-month deprecation period. @see [#385](https://github.com/heremaps/here-data-sdk-typescript/pull/385).
```

Sorry for this. I thought github creates automatic links to PRs as it does inside review comments.